rb-files/interface.rb

```diff
@@ -0,0 +1,48 @@
+require_relative './app'
+
+class Interface
+  def initialize(app)
+    @app = app
+  end
+
+  def start
+    puts 'Welcome to School Library App!'
+    loop do
+      display_menu
+      user_choice = gets.chomp.to_i
+      handle_user_choice(user_choice, @app)
+    end
+  rescue Interrupt
+    warn "\nProgram exited"
+    exit 130
+  end
+end
+
+def display_menu
+  puts "\nPlease choose an option by entering a number:"
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def handle_user_choice(choice, app)
+  menu_options = {
+    1 => :list_books,
+    2 => :list_people,
+    3 => :create_person,
+    4 => :create_book,
+    5 => :create_rental,
```

Comment:
- [ ] This is a bad example of a single-responsibility rule. All requests for input and input validation in your methods should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input. Kindly remove them from your functions in the app.rb and move them to this file. You can check this lesson to remember them: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid

Reply:
@tugsem I don't understand your comment. Please elaborate it..